source/basic.tex

```diff
@@ -546,17 +546,17 @@
 \begin{itemize}
 \item class type\iref{class},
 \item enumeration type\iref{dcl.enum},
-\item inline function with external linkage\iref{dcl.inline},
-\item inline variable with external linkage\iref{dcl.inline},
+\item inline function or variable with external linkage\iref{dcl.inline},
```

Comment:
This _could_ be read as "either an inline function (with external/internal/no linkage), or a variable (inline or not) with external linkage". Separate bullets avoids that ambiguity.

Reply:
@zygoloid said the same thing and approved of just relocating the cross-reference (as has now been done).